.github/workflows/release.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches: [ master, add/gh-action-prod ]
 
```

Comment:
- Consider providing more specific error messages or handling for cases where commit messages or descriptions do not match the expected patterns.